src/dfeed/web/captcha/dcaptcha.d

```diff
@@ -52,7 +52,7 @@ final class Dcaptcha : Captcha
 			`<input name="dcaptcha_response_field"></input>` ~
 			`<p><b>` ~ _!`Hint` ~ `</b>: ` ~ challenge.hint ~ `</p>` ~
 			`<p>` ~ _!"Is the CAPTCHA too hard?\nRefresh the page to get a different question,\nor ask in the %s#d IRC channel on Freenode%s."
-				.replace("\n", `<br>`)
+				.replace(`\n`, `<br>`)
```

Comment:
This doesn't look right. I'm guessing this fixes the Turkish string but breaks the English string.

Reply:
Oh? I will check but I think this is right because the original strings are defined with back-ticks (at least in the turkish.d file), meaning that they don't have \n "characters". But this string tries to replace the \n character. The change makes it replace the two character sequence \ and n.